README.md

```diff
@@ -13,17 +13,24 @@ Currently, only Java 8 with the build tool sbt is supported. We hope to increase
 compatibility with more Java language versions and build tools as the project
 evolves.
 
-| Language version | Support |
-| ---------------- | ------- |
-| Java 7           | ❌      |
-| Java 8           | ✅      |
-| Java 11+         | ❌      |
+| Language version | Support            |
+| ---------------- | ------------------ |
+| Java 7           | ❌                 |
+| Java 8           | ✅                 |
+| Java 11          | ✅                 |
+| Java 12          | Untested, may work |
+| Java 13          | Untested, may work |
+| Java 14          | Untested, may work |
+| Java 15          | Untested, may work |
```

Comment:
Works on Java 15, just not formally in the CI test, so Im assuming youre leaving it out for that reason?

Reply:
Strictly speaking, the "Java 8" row should have the same message as Java 15. The next item on my agenda is to get cross-java CI working so this is only temporary.